src/Xrefcheck/Config/Default.hs

```diff
@@ -67,6 +67,9 @@ scanners:
     #
     # This affects which anchors are generated for headers.
     flavor: #s{flavor}
+
+  # Whether copy-paste check is enabled globally.
+  copyPasteCheckEnabled: True
```

Comment:
By being consistent with the aims of this project in the `README` file, I think that it should be `false` by default, but I also think that then the user may never notice about the feature.

To consider other options, we could also try to add an intermediate configuration setting:

```
copyPasteCheck: (error|warning|disabled)
```

where:

* `error` shows them as errors.
*  `warning` shows them as warnings, which do not make the program to exit with error code.
*  `disabled` disables the feature.

and the default would be `warning`.

In the case of going for the different `dump-config` strictness modes, I guess that this mode argument should be mandatory. Otherwise, we would be again discussing what should be the default, and the weak one could make the user to miss some features.

Reply:
> warning shows them as warnings, which do not make the program to exit with error code.

Hm do we need this? I think that the user can look at the error decriptions and decide whether to just ignore those links, or to fix them immediately, it seems that showing them as warnings is rather useless. So, I would agree with @Martoon-00, we can just provide some modes of the default config, and using true/false for different modes perfectly suites the idea.

Reply:
If I understood correctly, the doubt was in deciding whether to enable this check by default, which could produce false positives in CI, or to disable it and possibly make the user to never know about the feature.

I think that providing modes of default config can preserve the same problem, because then we would have to decide whether to set a strong one as the default and produce false positives in CI, or to set a weak one and make the user to never know about some features, although it would be easier to manage.

The point in my suggestion was to consider an intermediate option that does not produce false positives in CI, but keeps showing messages that the user can see in order to know about the feature.

Reply:
Hmhmhm :thinking: :thinking:  Both your points make sense to me, I'll need some time to digest this and produce my thoughts.